spikes/npgsql-vanilla/Makefile

```diff
@@ -0,0 +1,21 @@
+DOTNET_FRAMEWORK=net5.0
+
+help:
+	@echo "Please choose one of the following targets: start-cratedb, build, run"
+	@exit 2
+        
+start-cratedb:
+	docker run -it --rm --publish=4200:4200 --publish=5432:5432 crate:4.5.1
+
+build:
+	dotnet build
+
+run:
+	dotnet run --framework=${DOTNET_FRAMEWORK}
+
+test:
+	dotnet test --framework=${DOTNET_FRAMEWORK} --verbosity=minimal
+	@#dotnet test --verbosity=normal
```

Comment:
```suggestion
```

Reply:
I'd  remove the whole Makefile, no need to build a custom make based dsl around the dotnet command. Imho it obfuscates what is happening and doesn't offer any advantage. Dotnet developers will know how to use the `dotnet` cli, they won't know what our custom make targets are and will have to learn something extra if they look at this.

But I'll leave the decision to you.

Reply:
Thank you. I moved remaining "notes" from the Makefile into the corresponding README.rst instead and removed the Makefile afterwards, see fc64076aa.